api/rest/v1/find_entities.md

```diff
@@ -0,0 +1,117 @@
+</div>
+
+<div markdown="span" class="alert alert-warning" role="alert">
+    <span class="material-icons md-16">info </span><b>See Also:</b><br />
+    For querying multiple entities, see the [bulk version](/api/rest/v1/bulk/find/entities) of this endpoint.
+</div>
+
+## Request
+
+GET Request
+{: .api-header}
+
+<div class="api-signature">
+URL:
+https://api.datacommons.org/v1/find/entities?key={your_api_key}&description={entity_description}
+</div>
+<script src="/assets/js/syntax_highlighting.js"></script>
+
+### Path Parameters
+
+There are no path parameters for this end point.
+
+### Query Parameters
+
+| Name                                                     | Type   | Description                                                                                                                                                     |
+| -------------------------------------------------------- | ------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| key <br /> <required-tag>Required</required-tag>         | string | Your API key. See the [page on authentication](/api/rest/v1/getting_started#authentication) for a demo key, as well as instructions on how to get your own key. |
+| description <br /> <required-tag>Required</required-tag> | string | Description of the entity. Typically the name of the place.                                                                                                                                         |
+| type <br /> <optional-tag>Optional</optional-tag>        | string | The type of entity, specified as a DCID. Common values are "Country", "State", "County", "City".                                                                         |
+{: .doc-table }
+
+## Response
+
+The response looks like:
+
+```json
+{
+  "dcids": ["dcid"]
+}
+```
+{: .response-signature .scroll}
+
+### Response fields
+
+| Name        | Type   | Description                                                                                            |
+| ----------- | ------ | ------------------------------------------------------------------------------------------------------ |
+| dcids       | list   | DCIDs matching the description you provided. If no matches are found, this field will not be returned. |
+{: .doc-table}
+
+## Examples
+
+### Example 1: Find the DCID of a place
+
+This queries for the DCID of "_Georgia_". Notice that specifying "_Georgia_" without specifying `type` returned the DCID of the US state.
+
+Request:
+{: .example-box-title}
+
+```bash
+$ curl --request GET --url \
+'https://api.datacommons.org/v1/find/entities?key=AIzaSyCTI4Xz-UW_G2Q2RfknhcfdAnTHq5X5XuI&description=Georgia'
```

Comment:
nit: put "key=xxx" at the end of examples. Ditto for examples elsewhere.

Reply:
Good point. Fixed!